src/test/scala/lms/thirdparty/test_cuda.scala

```diff
@@ -311,7 +325,7 @@ class CudaTest extends TutorialFunSuite {
         }
       }
     }
-//    check("softmax", driver.code, "cu")
+    // check("softmax", driver.code, "cu")
```

Comment:
use check file here?

Reply:
Yes, I need to implement the PyTorch tests as you did with embedding